src/lib/authui/moreauthcomps/main_dialog.rb

```diff
@@ -0,0 +1,458 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2016 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact SUSE Linux GmbH.
+#
+# ------------------------------------------------------------------------------
+
+require 'yast'
+require 'auth/authconf.rb'
+require 'authui/moreauthcomps/edit_realm_dialog'
+Yast.import 'UI'
+Yast.import 'Label'
+
+module MoreAuthComps
```

Comment:
Seeing "More" in a module name makes me feel uneasy. It is too likely to turn into a MiscellaneousUnstructuredMessyDump IMHO. Why cannot we simply have an AuthenticationComponents module? (or AuthComponents, since the Auth abbreviation is well established already)